mods/ts/weapons/energyweapons.yaml

```diff
@@ -124,6 +126,7 @@ CyCannon:
 		RangeLimit: 8c0
 		TerrainHeightAware: true
 	Warhead@1Dam: SpreadDamage
+		ValidTargets: Ground, Trees
```

Comment:
This is missing on the weapon to allow force-firing.

Reply:
And also from the (Warhead@2Eff) `CreateEffect` warhead (which is inherited). It seems to be missing from Grenade and Bomb too. The super weapons also miss it on that and on sound etc. Please double check the other weapons for missing `ValidTargets` on effect warheads.